acceptance-tests/dsl/src/main/java/org/hyperledger/besu/tests/acceptance/dsl/transaction/privacy/PrivacyRequestFactory.java

```diff
@@ -16,12 +16,43 @@
 
 import static java.util.Collections.singletonList;
 
+import org.hyperledger.besu.enclave.types.PrivacyGroup;
+import org.hyperledger.besu.ethereum.api.jsonrpc.internal.privacy.parameters.CreatePrivacyGroupParameter;
+
+import java.util.HashMap;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import org.assertj.core.util.Lists;
 import org.web3j.protocol.Web3jService;
 import org.web3j.protocol.besu.Besu;
 import org.web3j.protocol.core.Request;
 import org.web3j.protocol.core.Response;
 
 public class PrivacyRequestFactory {
+
+  public static class GetPrivacyPrecompileAddressResponse extends Response<String> {}
```

Comment:
I am slightly torn between using inner static class vs declared them as separate source files.

Reply:
Felt a bit excessive to extract the empty class to a separate source file.